README

```diff
@@ -108,6 +108,10 @@ Mandriva:
 	Mandriva has integrated fio into their package repository, so installing
 	on that distro should be as easy as typing ``urpmi fio``.
 
+Arch Linux:
+        Arch Linux package is provided under community.
```

Comment:
Can we make it "community sub-repository" then? I agree that if you aren't familiar with Arch, then the current version reads a bit funny.